src/HW/client/src/components/DoneTasks/DoneTasksConnector.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import DoneTasks from "./DoneTasks";
+import {getSearchInput, getDoneItems } from "../../redux/selectors/itemsEntitiesSelectors";
+
+
+const mapStateToProps = (state, ownProps) => {
+    
+    const searchInputValue = getSearchInput(state);
+   
+    const doneItems = getDoneItems(state)
+    return {searchInputValue,doneItems };
+  };
+  
+  const mapDispatchToProps = (dispatch, ownProps) => {
```

Comment:
if you dont need to bind anything just dont add the function. then call connect only with mapStateToProps